### PR TITLE
Updating Benefit questions for en and cy.

### PR DESCRIPTION
**Pull request by zaparka**
Updating the look and copy of benefits page.

### CODE REVIEW

File: features/step_six.feature [hunk 1]
```diff
@@ -7,19 +7,25 @@ Feature: Step six page
 
   Scenario: Displays step number
     Then I should see step 6 of 20
-  
+
   Scenario: Displays header
     Then I should see 'Do you receive any of the following benefits?' header
 
   Scenario: Displays benefits list
     Then I should see the benefits list:
```
Comment (AlexaVB) [inline features/step_six.feature]: Maybe have one scenario for displays instruction copy:
'If you are receiving one of the benefits listed in column 1, select ‘Yes’.                                                
There are some benefits with similar names listed in column 2. If you are receiving one of these benefits, select 'No'.   
If you are not receiving any of benefits listed on this page, select 'No'. 
Another scenario for benefits table.'

File: features/step_six.feature [hunk 2]
```diff
@@ -7,19 +7,25 @@ Feature: Step six page
 
   Scenario: Displays step number
     Then I should see step 6 of 20
-  
+
   Scenario: Displays header
     Then I should see 'Do you receive any of the following benefits?' header
 
   Scenario: Displays benefits list
```
Comment (AlexaVB) [inline features/step_six.feature]: Maybe call this something like 'Displays instruction bullet points' or 'Displays instructions'

File: features/step_six.feature [hunk 3]
```diff
@@ -7,19 +7,25 @@ Feature: Step six page
...
-    | Income Support                                                                          |
-    | Universal Credit (and you’re earning less than £6,000 a year)                           |
-    | Pension Credit (Guarantee Credit)                                                       |
-    | Scottish Civil Legal Aid (not Advice and Assistance or Advice by Way of Representation) |
+    | benefit                                                                                                                   |
+    | If you are receiving one of the benefits listed in column 1, select ‘Yes’.                                                |
+    | There are some benefits with similar names listed in column 2. If you are receiving one of these benefits, select 'No'.   |
+    | If you are not receiving any of benefits listed on this page, select 'No'.                                                |
+
+    Then I should see the benefits table:
```
Comment (AlexaVB) [inline features/step_six.feature]: Maybe have this as a separate scenario 'Displays benefits table' or something

File: features/support/page_objects/step_six_page.rb [hunk 4]
```diff
@@ -3,6 +3,7 @@ class StepSixPage < BasePage
     element :benefits_text, '.text', text: 'contact the Department for Work and Pensions'
     element :benefit, '.text'
     elements :li, 'li'
+    elements :td, 'td'
```
Comment (garytaylor) [inline features/support/page_objects/step_six_page.rb]: The page object pattern is supposed to provide a nice interface to the test suite that isolates it from the implementation's details like td's this etc...  So having a method called :td just to give you all td's seems to be defeating the object of it.  But, there seems to be a pattern set down already with the :li above it.

Comment (AlexaVB) [reply features/support/page_objects/step_six_page.rb]: Yeah I know. The problem is the element doesn't have a descriptive name. If you change it to one then you add a layer of ambiguity. I think it's fine as td

**Comment by zaparka** [comment]
@AlexaVB changes applied.

**Comment by AlexaVB** [comment]
Perfect!

**Pull request merged**